clients/vcva/config.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+##################################################
+
+ses_address="10.23.82.167"
```

Comment:
Replace `ses` wit `srs`
Remove the values from the file, it's not allowed to hold ips and passwords on github.